pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -173,7 +176,9 @@ def process_circuits(
                 options.resilience_level = 0
                 options.execution.shots = n_shots
                 options.simulator.noise_model = self._noise_model
-                options.seed_simulator = kwargs.get("seed")
+                options.seed_simulator = seed
+                if kwargs.get("seed_auto_increase") and type(seed) is int:
+                    seed += 1
```

Comment:
```suggestion
                seed += 1
```

Reply:
Done in [85489ef](https://github.com/CQCL/pytket-qiskit/pull/197/commits/85489ef983300e4739976ba68048e55e5ceb932a)

Reply:
I needed to keep the type check for mypy. (and to make sure I am not incrementing when the seed is set to none)